src/main/datatypes/MQ2GroupType.cpp

```diff
@@ -40,6 +40,25 @@ enum class GroupMembers
 	Injured,
 	LowMana,
 	Cleric,
+	Shaman,
+	Warrior,
+	Paladin,
+	Ranger,
+	Shadowknight,
+	Druid,
+	Monk,
+	Bard,
+	Rogue,
+	Necromancer,
+	Wizard,
+	Mage,
+	Magician,
+	Enchanter,
+	Beastlord,
+	Berserker,
```

Comment:
I cannot accept this change as implemented. This is not a pattern I want to propogate -- the same block of code down below is copied 13 times.

`Cleric` is a huge anti pattern that eqmule threw into this because of a personal use case with no bearing on the overall design (case in point).

If it is really important to have this functionality, we need to design a new interface for it that isn't a copy pasta. Perhaps instead we need something like  `MemberByClass`